Dockerfile

```diff
@@ -16,7 +16,7 @@ COPY ./internal/csl ./internal/csl
 COPY [ "go.mod", "go.sum", "./" ]
 RUN GOOS=js GOARCH=wasm go build -o dist/csl.wasm ./cmd/cslwasm
 RUN brotli -f dist/csl.wasm
-RUN cp $(go env GOROOT)/misc/wasm/wasm_exec.js .
+RUN cp $(go env GOROOT)/lib/wasm/wasm_exec.js .
```

Comment:
this seems to have been moved:

https://github.com/golang/go/blob/go1.25.0/misc/wasm/wasm_exec.html

https://github.com/golang/go/blob/go1.17/misc/wasm/wasm_exec.html